lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.tsx

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { FunctionComponent, ReactElement } from 'react';
+
+import { ThreatLevelCategory, categoryByPolicyThreatLevel } from '../../util/threatLevels';
+import { Props, propTypes } from './types';
+export { Props } from './types';
+
+import './NxPolicyViolationIndicator.scss';
+import NxFontAwesomeIcon from '../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+
+const NxPolicyViolationIndicator: FunctionComponent<Props> =
+  function NxPolicyViolationIndicator({ threatLevelCategory, policyThreatLevel, children }): ReactElement<Props> {
+    const category: ThreatLevelCategory =
+          threatLevelCategory ? threatLevelCategory :
+          policyThreatLevel != null ? categoryByPolicyThreatLevel[policyThreatLevel] : 'unspecified',
+
+        className = `nx-policy-violation-indicator nx-policy-violation-indicator--${category}`,
+        label = `threat level ${category}`;
+
+    return (
+      <div className={className}>
+        <div className="nx-policy-violation-indicator__badge">
+          <NxFontAwesomeIcon aria-hidden={false}
+                             aria-label={label}
+                             icon={faExclamationCircle}/>
+        </div>
+        <div className="nx-policy-violation-indicator__text">
+          <span>{children || category}</span>
```

Comment:
Unnecessary span?  Additionally would the `.nx-policy-violation-indicator__badge` and `nx-policy-violation-indicator__text` be better as spans than divs?

Reply:
> Unnecessary span? Additionally would the `.nx-policy-violation-indicator__badge` and `nx-policy-violation-indicator__text` be better as spans than divs?

Removing the span: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/3da1f0356c771ee48cfe2d33ec036362524de1c8